modules/core/src/parallel.cpp

```diff
@@ -455,6 +455,29 @@ void cv::parallel_for_(const cv::Range& range, const cv::ParallelLoopBody& body,
     }
 }
 
+#if __cplusplus >= 201103L
+class ParallelLoopBodyLambdaWrapper : public cv::ParallelLoopBody
```

Comment:
I believe we can put this into header files (without requirement for new CV_EXPORTS functions)